BanchoSharp/Multiplayer/MultiplayerLobby.cs

```diff
@@ -417,12 +417,19 @@ private void UpdatePlayerInformation(string banchoResponse)
 		// Bancho may send extra player info after the name, for example "[Host / HardRock]", after the 16
 		// character player name bit.
 		string? playerInfo = banchoResponse.Length > (playerNameBegin + 16) ? banchoResponse[(playerNameBegin + 16)..] : null;
-
+		
+		// Find the digits from "/u/" to where the name begins
+		int playerId = int.Parse(banchoResponse[banchoResponse.IndexOf("/u/", StringComparison.Ordinal)..(playerNameBegin-1)].Where(char.IsDigit).ToArray());
+		
```

Comment:
Put this in a `.TryParse` instead, if for some reason it doesn't work we don't want to crash.

Reply:
I mean sure, I wasn't considering it since we do `.Parse` at multiple other places in the parsing code, and this should **never** be invalid in the first place, I'll be ready to eat my own words if that were to happen.